playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -586,10 +586,22 @@ EDXAPP_CODE_JAIL_LIMITS:
 
 # Set the number of workers explicitly for lms and cms
 # Should be set to
-# EDXAPP_WORKERS:
-#   lms: <num workers>
-#   cms: <num workers>
+EDXAPP_WORKERS:
+  lms: 14
+  cms: 14
```

Comment:
I know we'll be overriding this in OCIM, but I think in general for any instance we don't need as many CMS workers as LMS, since the CMS will likely never have as much traffic as LMS.
Can we decrease this to match our OCIM config?

Reply:
@kaizoku I've updated this, can you take a look? Thanks.

Reply:
Looks good, thanks @xirdneh